ompi/mpi/fortran/mpif-h/register_datarep_f.c

```diff
@@ -89,28 +90,28 @@ static int extent_intercept_fn(MPI_Datatype type_c, MPI_Aint *file_extent,
 /* Data structure passed to the intercepts (see below).  It is an OPAL
    list_item_t so that we can clean this memory up during
    MPI_FINALIZE.  */
-typedef struct intercept_extra_state {
+typedef struct ompi_intercept_extra_state {
```

Comment:
I do agree the specific string ```opal_``` is weird though... not sure what to do about that. I don't really want this to turn into a manual name change spanning all those files. Is there a var that indicates ```ompi_ / opal_ / orte_``` etc```?